sidebars.js

```diff
@@ -37,6 +37,7 @@ module.exports = {
     {
       type: "category",
       label: "Release notes",
+      link: {type:"doc", id:"getting-started/release-notes"},
```

Comment:
Try renaming `index.md` in the `release-notes` folder back to `release-notes-overview.md` and then update the `link` attribute like so:

```suggestion
      link: {type:"doc", id:"getting-started/release-notes/release-notes-overview"},
```